comp/otelcol/extension/README.md

```diff
@@ -0,0 +1,57 @@
+# Datadog Extension
+
+The datadog extension component allows inclusion of otel-agent data in the datadog-agent [flare](https://docs.datadoghq.com/agent/troubleshooting/send_a_flare/?tab=agent). A flare can be triggered by the core agent process, or remote-config.
```

Comment:
We should also add how the extension provides the relevant metadata for `otel-agent` configuration and inventory tracking in Fleet Automation. This metadata is periodically collected by the core-agent which then submits that data to the backend..

Reply:
Added, can you please take a look ?